cmd/rosetta-dispatcher-server/main.go

```diff
@@ -0,0 +1,297 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/labstack/echo/v4"
+	"github.com/ziflex/lecho/v2"
+	"io/ioutil"
+	"net/http"
+	url2 "net/url"
+	"os"
+	"os/signal"
+	"sort"
+	"time"
+
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/optakt/flow-dps-rosetta/service/identifier"
+	"github.com/rs/zerolog"
+	"github.com/spf13/pflag"
+)
```

Comment:
```suggestion
import (
	"bytes"
	"context"
	"encoding/json"
	"errors"
	"fmt"
	"io/ioutil"
	"net/http"
	"net/url"
	"os"
	"os/signal"
	"sort"
	"time"

	"github.com/labstack/echo/v4"
	"github.com/labstack/echo/v4/middleware"
	"github.com/rs/zerolog"
	"github.com/spf13/pflag"
	"github.com/ziflex/lecho/v2"
	
	"github.com/optakt/flow-dps-rosetta/service/identifier"
)
```

Imports should be grouped consistently: standard library, then external deps, then `onflow` deps if any, then internal deps.

Also no need for an import alias (`url2`) on the `net/url` package.